orm/src/test/java/org/hibernate/search/test/query/dsl/DSLTest.java

```diff
@@ -351,6 +354,22 @@ public void testBoolean() throws Exception {
 		assertEquals( 1, results.size() );
 		assertEquals( "January", results.get( 0 ).getName() );
 
+		//must + disable scoring
```

Comment:
Just a style-thing: I'd rather create a dedicated test method for this (also for the other logical groups of assertions within that method btw.). That way e.g. a test failure on CI makes more apparent which usage is causing the error. The inline comments give a good indication what should be separate methods.